### DISCUSSION TITLE
Dynamic level 3 headers unfolding

### DISCUSSION THREAD

**Discussion by ShutdownRepo** [GitbookIO/community#75] [Feature Requests]
### What's your feature/idea?

On the right panel, the table of content is limited to level 1 and level 2 headers.
I'd be awesome to have the TOC unfold dynamically when scrolling. For instance, let's say the user clicks or scrolls to a level 2 header on top of a few level 3 headers, the TOC would unfold on that part only and show the level 3 headers.
This is similar to what hackmd has (example : https://hackmd.io/c/book-example/%2Fs%2Ffeatures) where the TOC only show level 1 headers, and unfold dynamically to show level 2. I'd love something similar with level 1 and 2 always unfolded (like it currently is), and level 3 (and 4?) unfolding dynamically

### What's your desired outcome?

I described my outcome idea above, but whatever outcome allows to have level 3 headers (and 4?) in the TOC is great.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by ShutdownRepo** [comment]
Any news on this 😊 ?

**Reply by taszap** [reply]
Hey @ShutdownRepo thanks! At the moment it's not planned - we are focussing our efforts on different aspect of the product and I'd rather be transparent with you. We will definitely post an update if there is movement on this
